src/Field/DataField.php

```diff
@@ -58,6 +60,7 @@ public function __construct(
         public readonly string $valueTag = '',
         public readonly array|Closure $valueAttributes = [],
         public readonly bool $encodeValue = true,
+        public readonly bool $isVisible = true,
```

Comment:
```suggestion
        public readonly bool $visible = true,
```

For consistency with parameter `$visible` in `DataColumn` constructor.